### DISCUSSION TITLE
Lane Markings

### DISCUSSION THREAD

**Discussion by andy-96** [bdd100k/bdd100k#97] [Q&A] [answered]
Hi all!

Thanks a lot for the great work!
I wanted to ask whether there is an updated version of the lane labels or are all the lane labels in "Labels"?
Unfortunately, the labels are sometimes quite bad, especially when it comes to curves.

Thanks a lot!

**Comment by fyu** [comment, marked as answer]
We don't have updates for lane marking at this point. It may be worth noting that the curves are usually labeled by bezier curves.
